.gitignore

```diff
@@ -5,4 +5,7 @@ target/
 
 # TypeScript builds
 dist/js/*.js
-dist/js/*.map
\ No newline at end of file
+dist/js/*.map
+
+# code workspace lol
+**/*.code-workspace
```

Comment:
Probably don't wanna modify that in the same PR :P

Reply:
I would still suggest reverting the changes in the `.gitignore` ^^

Reply:
I thought i had already, ill do it rq